etl/app/server/database.js

```diff
@@ -0,0 +1,184 @@
+import pg from 'pg';
+
+import { logger as log } from './utilities/logger.js';
+import * as create from './queries/create.js';
+
+const { Pool } = pg;
+
+const dbName = 'expert_query';
```

Comment:
```suggestion
const dbName = process.env.DB_NAME ?? 'expert-query';
```

Switch this to pull from `.env`.